operator/v1/types_kubeapiserver.go

```diff
@@ -35,6 +35,18 @@ type KubeAPIServer struct {
 
 type KubeAPIServerSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+
+	// eventTTLMinutes specifies the amount of time that the events are stored before being deleted.
+	// This setting is allowed between 30 minutes minimum up to 6h (360 minutes).
+	//
+	// When omitted this means no opinion, and the platform is left to choose a reasonable default, which is subject to change over time.
+	// The current default value is 3h (180 minutes).
+	//
+	// +kubebuilder:default:=180
+	// +kubebuilder:validation:Minimum=30
+	// +kubebuilder:validation:Maximum=360
```

Comment:
Do we have users that need to _increase_ the event TTL above 3h? Choosing 3h as the maximum would preserve the upper bound when it comes to benchmarking and publishing supported cluster sizes.